desc/compute/_metric.py

```diff
@@ -1823,3 +1823,74 @@ def _gradtheta(params, transforms, profiles, data, **kwargs):
 def _gradzeta(params, transforms, profiles, data, **kwargs):
     data["|grad(zeta)|"] = jnp.sqrt(data["g^zz"])
     return data
+
+
+@register_compute_fun(
+    name="gbdrift",
+    label="\\mathrm{gradB-drift}",
+    units="~",
+    units_long="None",
+    description="Binormal component of th geometric part of the gradB drift"
+    + " used for local stability analyses,"
+    + " energetic particle proxy,"
+    + " effective field ripple",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="rtz",
+    data=["|B|", "b", "grad(alpha)", "grad(|B|)"],
+)
+def _gbdrift(params, transforms, profiles, data, **kwargs):
+    data["gbdrift"] = (
+        1
+        / data["|B|"] ** 2
+        * dot(data["b"], cross(data["grad(|B|)"], data["grad(alpha)"]))
+    )
+    return data
+
+
+@register_compute_fun(
+    name="cvdrift",
+    label="\\mathrm{curvature-drift}",
+    units="~",
+    units_long="None",
+    description="Binormal component of th geometric part of the curvature drift"
+    + " used for local stability analyses,"
+    + " energetic particle proxy"
+    + " effective field ripple",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="rtz",
+    data=["p_r", "psi_r", "|B|", "gbdrift"],
+)
+def _cvdrift(params, transforms, profiles, data, **kwargs):
+    mu_0 = 4 * jnp.pi * 1.0e-7
```

Comment:
elsewhere we just do `from scipy.constants import mu_0` at the top of the file

Reply:
Done!